client/my-sites/stats/stats-period-header/style.scss

```diff
@@ -35,3 +36,17 @@
 		margin-top: 0;
 	}
 }
+
+// styling for both interval dropdown and date control picker drop down buttons inside the header
+.components-dropdown > .components-button {
```

Comment:
This rule feels a bit too generic and could potentially bleed out. I'd personally prefer to see these rules in the closest parent (ie: `stats-date-control`). Something like this:

```
.stats-date-control {
    // some CSS bits
	.components-button {
        // CSS bits specific to buttons inside the stats-date-control container
		background: red;
	}
}
```